src/Akka.Persistence.Azure/Journal/AzureTableStorageJournal.cs

```diff
@@ -863,5 +1007,15 @@ private async Task<IEnumerable<string>> GetAllPersistenceIds()
                 return _allPersistenceIds.Add(persistenceId);
             }
         }
+        private void NotifyPersistenceIdChange(string persistenceId)
+        {
+            if (_persistenceIdSubscribers.TryGetValue(persistenceId, out var subscribers))
```

Comment:
This does not make sense.
who are you notifying? 
who are the subscribers? 
why are there multiple types of subscribers? 
why are you segmenting the subscribers based on persistent ids?
why are you notifying that there is a new persistenceId being created to a group of subscribers in a non existent persistenceId subscriber group?